R/add_missing_events.R

```diff
@@ -0,0 +1,325 @@
+#Author: Paigan Aspinall
+#Date & version: 15FEB2024 V1.0.1
+#R version: 4.2.2
+
+#Unblinded weekly report script
+
+#Set your API token
+token <- '86CDF603FECAD55D05B27F6D3FA5ACEC'
```

Comment:
Security vulnerability. This is also now in the git audit trail. If you can, you should refresh whatever token this is so that this one is no longer able to retrieve data.

Reply:
API token updated.

Reply:
And correct file added now.